spec/openapi.yaml

```diff
@@ -7711,6 +7711,309 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+  /jobs-api/v3/projects/{projectId}/custom-fields:
+    get:
+      summary: Find custom fields for specified project
+      description: >-
+        Get all custom fields for specified project.
```

Comment:
Lest change to `Returns custom fields assigned to the project. Order is the one that was specified during assignment operation.`